tasks/e2e.sh

```diff
@@ -53,15 +58,22 @@ set -x
 cd ..
 root_path=$PWD
 
+npm install
+
+# If the node version is < 4, the script should just give an error.
+if [ `node --version | sed -e 's/^v//' -e 's/\..\+//g'` -lt 4 ]
+then
+  cd $temp_app_path
+  node "$root_path"/packages/create-react-app/index.js test-node-version && exit 1 || exit 0
```

Comment:
We should verify that it prints a nice message instead of failing with e.g. a parse error.